File: javascript01/step11/test02_2.js
/* 데이터 추가 명령: insert
insert into 테이블명(컬럼명, 컬렁명, ...)
values(값, 값, ...)
*/
const mysql = require('mysql')

const connection = mysql.createConnection({
  database: 'webappdb',
  user : 'webapp',
  password: '1111'
})

connection.connect()

connection.query('insert into memb(name, email, tel, pwd)' +
                 'values("aaa", "aaa2@test.com", "111-1111", password(1111))', function (error, results){  // MySQL에서 암호로 데이터 베이스에 저장하는 함수가 password()
  console.log('결과를 가져왔음!')
  if (error) throw error;  //에러를 콘솔창으로 출력한다고 보면된다.

  console.log(results)
}
)

connection.end()
console.log('MySQL 서버와의 연결끊기가 예약되었음!')

File: java01/src/step20/Test02_2.java
/* JDBC 프로그래밍 : Statement 객체 - INSERT 실행*/

package step20;

import java.sql.DriverManager;

public class Test02_2 {

  public static void main(String[] args) throws Exception {
    System.out.println("step20 Test02_2 실행중");System.out.println();
    
    com.mysql.jdbc.Driver mysqlDriver = new com.mysql.jdbc.Driver();
    
    DriverManager.registerDriver(mysqlDriver);
    
    java.sql.Connection con = DriverManager.getConnection(
        "jdbc:mysql://localhost:3306/webappdb", 
        "webapp",
        "1111");
    
    java.sql.Statement stmt = con.createStatement();
    
    int count = stmt.executeUpdate("insert into memb(name,tel,email,pwd) "
        + "values('하하하', '1212-1313', 'haha@test.com', password('1111'))");
    
    System.out.println(count);
    
    stmt.close();
    con.close();
    
  }

}
